src/g_text.c

```diff
@@ -584,9 +584,7 @@ static void gatom_set(t_gatom *x, t_symbol *s, int argc, t_atom *argv)
     if (x->a_atom.a_type == A_FLOAT)
     {
         x->a_atom.a_w.w_float = atom_getfloat(argv);
-        changed = ((x->a_atom.a_w.w_float != oldatom.a_w.w_float));
-        if (isnan(x->a_atom.a_w.w_float) != isnan(oldatom.a_w.w_float))
```

Comment:
This clause is a no-op, because if either value were a NaN, then the `!=` check on the previous line would already result in `true`.

Reply:
BTW, this is not a no-op when compiling with -ffast-math because the compiler might generate assembly which can give wrong results for comparisons involving NaNs (I've been hit by this personally: https://stackoverflow.com/questions/57894032/floating-point-comparison-with-quiet-nan-on-gcc-with-ffast-math-always-yields)

But with the `memcmp` it becomes obsolete, of course.

Reply:
TIL! Thanks for the info.